default.nix

```diff
@@ -46,7 +54,6 @@ stdenv.mkDerivation rec {
   ] else []) ++ (if buildDoc then [
 
     # Sphinx doc dependencies
-    ocamlPackages.lablgtk
```

Comment:
Unrelated?

Should that be deleted or just moved elsewhere?

Reply:
You’re right, this is an unrelated (minor) cleaning. This line is a duplicate of 5 lines above (a dependency of CoqIDE).

Reply:
Concerning `fetchTarball` I’ve no strong opinion. I just pushed a change.

Reply:
Well, since you were bikeshedding to begin with :-p .

Reply:
Note that if we bikeshed long enough, a channel will be ready with all dependencies, so that I the URL could be updated…

Reply:
That's a good strategy.

That, or merging today.